ngraph/core/src/op/add.cpp

```diff
@@ -19,35 +19,9 @@
 #include "ngraph/runtime/host_tensor.hpp"
 #include "ngraph/runtime/reference/add.hpp"
 
-NGRAPH_SUPPRESS_DEPRECATED_START
-
 using namespace std;
 using namespace ngraph;
 
-// ------------------------------- v0 ------------------------------------------
-
-constexpr NodeTypeInfo op::v0::Add::type_info;
-
-op::v0::Add::Add(const Output<Node>& arg0,
-                 const Output<Node>& arg1,
-                 const AutoBroadcastSpec& auto_broadcast)
-    : BinaryElementwiseArithmetic(arg0, arg1, auto_broadcast)
-{
-    constructor_validate_and_infer_types();
-}
-
-shared_ptr<Node> op::v0::Add::clone_with_new_inputs(const OutputVector& new_args) const
-{
-    check_new_args_count(this, new_args);
-    return make_shared<op::v0::Add>(new_args.at(0), new_args.at(1), this->get_autob());
-}
-
-bool op::v0::Add::visit_attributes(AttributeVisitor& visitor)
-{
-    BinaryElementwiseArithmetic::visit_attributes(visitor);
-    return true;
-}
-
 shared_ptr<Node> ngraph::operator+(const Output<Node>& arg0, const Output<Node>& arg1)
```

Comment:
After this change we will have + operator work based on opset1::Add.
@slyalin is it ok from backward compatibility standpoint?

Reply:
I propose to remove all *+-/ and etc. operators which allow to create nGraph ops. This approach cannot be versioned.